src/messages/search/MessageFlagsPredicate.cpp

```diff
@@ -0,0 +1,42 @@
+#include "messages/search/MessageFlagsPredicate.hpp"
+
+namespace chatterino {
+
+MessageFlagsPredicate::MessageFlagsPredicate(const QStringList &flags)
+    : flags_()
+{
+    // Check if any comma-seperated values were passed and transform those
+    for (const auto &entry : flags)
+    {
+        for (const auto &flag : entry.split(',', QString::SkipEmptyParts))
+        {
+            if (flag == "deleted")
```

Comment:
Just a nitpick, but you might add a check for `"disabled"` as well for all those nerds out there, who will use it instead of deleted.